src/content/docs/apm/agents/ruby-agent/getting-started/migration-9x-guide.mdx

```diff
@@ -0,0 +1,101 @@
+  | `disable_resque`                          | `instrumentation.resque`                  | `instrumentation.resque: disabled`                                                  |
+  | `disable_sinatra`                         | `instrumentation.sinatra`                 | `instrumentation.sinatra: disabled`                                                 |
+  | `disable_rake`                            | `instrumentation.rake`                    | `instrumentation.rake: disabled`                                                    |
+  | `disable_rake_instrumentation`            | `instrumentation.rake`                    | `instrumentation.rake: disabled`                                                    |
+  | `disable_typhoeus`                        | `instrumentation.typhoeus`                | `instrumentation.typhoeus: disabled`                                                |
+
+
+## Removed Support for Ruby 2.2 and 2.3 [#ruby_2223]
+
+  Ruby 2.2 and 2.3 are no longer supported by the ruby agent. In order to continue using the latest Ruby Agent version, please update to Ruby 2.4.0 or above.
+
+
+## Enabled Thread Tracing by Default [#thread_tracing]
+
+  The configuration option `instrumentation.thread.tracing` is now enabled by default. This will allow the agent to properly monitor code that occurs inside of threads. Work being done inside of threads that previously was not being recorded will now be recorded and associated with the transaction the thread was created in.
+  If you are currently using custom instrumentation to start a new transaction inside of threads, this may be a breaking change, as it will no longer start a new transaction if one already exists.
+
+
+## Removed Deprecated Instrumentation [#removed_gem_instrumentation]
+
+  Support for the following gems has been dropped. Metrics and events will no longer be created for these gems.
+    - Acts As Solr
+    - Authlogic
+    - DataMapper
+    - Rainbows
+    - Sunspot
+
+
+## Updated API method `set_transaction_name` [#set_transaction_name]
+
+  When the method `NewRelic::Agent.set_transaction_name` is called, it will now always change the name and category of the currently running transaction to what is passed in to the method. This is a change from how it functioned in previous agent versions. Previously, if `set_transaction_name` was called with a new transaction name and a new category that did not match the category that was already assigned to a transaction, neither the new name nor category would be saved to the transaction. 
+  If this method is being called in a situation in which it was previously ignored due to cateogry differences, this will now change the name and category of that transaction. 
+
+
+## Removed API methods [#removed_api_methods]
+
+  The following deprecated API methods have been removed.
+  - `NewRelic::Agent.disable_transaction_tracing` - Use `NewRelic::Agent.disable_all_tracing` instead
```

Comment:
Is it just one method or more methods are being removed but not listed yet?

Reply:
[Changed](https://github.com/newrelic/docs-website/pull/11408/commits/49d803baca6c63737258b4919314ece1808fe8dd) to singular "method"!